java/src/jmri/jmrix/marklin/cdb/CdBSystemConnectionMemo.java

```diff
@@ -0,0 +1,152 @@
+package jmri.jmrix.marklin.cdb;
+
+import java.util.Comparator;
+import java.util.ResourceBundle;
+
+import jmri.*;
+import jmri.jmrix.ConfiguringSystemConnectionMemo;
+import jmri.jmrix.DefaultSystemConnectionMemo;
+import jmri.util.NamedBeanComparator;
+
+import jmri.jmrix.marklin.MarklinTrafficController;
+//import jmri.jmrix.marklin.MarklinProgrammer;
+//import jmri.jmrix.marklin.MarklinProgrammerManager;
+import jmri.jmrix.marklin.MarklinSensorManager;
+import jmri.jmrix.marklin.MarklinThrottleManager;
+import jmri.jmrix.marklin.MarklinTurnoutManager;
+import jmri.jmrix.marklin.MarklinPowerManager;
+import jmri.jmrix.marklin.MarklinSystemConnectionMemo;
+
+/**
+ * Lightweight class to denote that a system is active, and provide general
+ * information.
+ * <p>
+ * Objects of specific subtypes are registered in the instance manager to
+ * activate their particular system.
+ *
+ * Based on work by Bob Jacobsen
+ *
+ * @author Kevin Dickerson Copyright (C) 2012
+ */
+public class CdBSystemConnectionMemo extends MarklinSystemConnectionMemo {
+
+    public CdBSystemConnectionMemo(MarklinTrafficController et) {
+        super("C", "CdB");
```

Comment:
@bobjacobsen 
Shouldn't this be `M` instead of `C`. `C` stands for [C/MRI serial](https://www.jmri.org/help/en/html/hardware/cmri/CMRI.shtml).

https://www.jmri.org/help/en/html/doc/Technical/Names.shtml

Reply:
Good catch.  Let me check for other places...